packages/components/icon/src/icon.tsx

```diff
@@ -14,27 +9,18 @@ import { useToken } from "@yamada-ui/use-token"
 import { cx, replaceObject, isUnit } from "@yamada-ui/utils"
 import type { FC, SVGAttributes } from "react"
 
-type IconOptions = {
-  /**
-   * The CSS `font-size` property.
-   *
-   * @deprecated Use `fontSize` instead.
-   */
-  size?: CSSUIProps["fontSize"]
-}
-
 export type IconProps = Omit<SVGAttributes<SVGElement>, keyof UIProps> &
   Omit<ThemeProps<"Icon">, "size"> &
-  UIProps &
-  IconOptions
+  UIProps
 
 /**
  * `Icon` is a general icon component that can be used in your projects.
  *
  * @see Docs https://yamada-ui.com/components/media-and-icons/icon
  */
 export const Icon = forwardRef<IconProps, "svg">((props, ref) => {
-  const [styles, { size, ...mergedProps }] = useComponentStyle("Icon", props)
+  const [styles, mergedProps] = useComponentStyle("Icon", props)
+
   let {
```

Comment:
Please change `let` to `const`.

Reply:
[c1449a8](https://github.com/yamada-ui/yamada-ui/pull/2537/commits/c1449a8f52780812c1522f8d85de494a1a9c1860)